www/js/services/services.js

```diff
@@ -0,0 +1,66 @@
+app.factory('Authorization', function($http){
+  var login = function(data) {
+    return $http({
+      method: 'POST',
+      url: 'api/login',
+      processData: false,
+      data: data,
+      headers:{'Content-Type':application/JSON}
+    })
+    .then(function(resp) {
```

Comment:
You use resp in the input and response in the actual function.  I'd prefer response if you could change them all to that.